iphone/Maps/UI/Search/TableView/MWMSearchCommonCell.xib

```diff
@@ -17,7 +17,7 @@
                 <autoresizingMask key="autoresizingMask"/>
                 <subviews>
                     <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="252" verticalCompressionResistancePriority="752" text="New York Cafe" textAlignment="natural" lineBreakMode="wordWrap" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="4FD-RE-ffF">
-                        <rect key="frame" x="16" y="12" width="220" height="20"/>
+                        <rect key="frame" x="16" y="12" width="220" height="20.5"/>
```

Comment:
Thanks! Are these coordinate changes necessary or can be removed from the PR?

Reply:
They're not needed. Seem to have been added by Xcode